### PR TITLE
[iframe] Added geolocation field.

### PR DESCRIPTION
**Pull request by Tjocksmurf**

### CODE REVIEW

**Comment by Tjocksmurf** [comment]
[iframe paragraph bundle (for hosted apps): add geolocation checkbox](https://trello.com/c/rg6C4fl2/373-iframe-paragraph-bundle-for-hosted-apps-add-geolocation-checkbox)

File: config/default/field.field.paragraph.para_h_content_iframe.field_h_allow_geolocation.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+field_name: field_h_allow_geolocation
+entity_type: paragraph
+bundle: para_h_content_iframe
+label: 'Allow geolocation'
+description: 'Is geolocation a feature of the application that is embedded in this iframe?'
+required: false
+translatable: false
+default_value:
+  -
+    value: 1
```
Comment (Benjen) [inline config/default/field.field.paragraph.para_h_content_iframe.field_h_allow_geolocation.yml]: @Tjocksmurf - I can't see us needing to use this functionality very often other than in very specific use cases. I would suggest we leave it off by default given someone could embed anything in an iframe.

**Review by Benjen — CHANGES_REQUESTED** [review]
Overall looks good. I did a bit of a tidy up in the iframe paragraph template as there was some stuff I had implemented previously which I didn't like.

I am also not sure that 'off' is a valid option for the "allow" property. So changed it to specifically disable access to geolocation.

I think we should not have geolocation enabled by default (see comment for more info).

File: config/default/field.field.paragraph.para_h_content_iframe.field_h_allow_geolocation.yml (hunk 1 above)
Comment (Tjocksmurf) [reply config/default/field.field.paragraph.para_h_content_iframe.field_h_allow_geolocation.yml]: I've changed it to disabled by default

**Review by Benjen — APPROVED** [review]

**Pull request merged**